shaka/src/public/ShakaPlayer.mm

```diff
@@ -136,35 +140,32 @@ @implementation ShakaPlayer
 
 // MARK: setup
 
-- (instancetype)initWithClient:(id<ShakaPlayerClient>)client {
+- (instancetype)initWithError:(NSError *__autoreleasing *)error {
```

Comment:
`__autoreleasing` for out parameters is the default behaviour, but the build script seems to be using a combination of parameters that causes an error in case this ownership is not explicitly specified:
```
../../../shaka/src/public/ShakaPlayer.mm:143:43: error: method parameter of type 'NSError *__autoreleasing *' with no explicit ownership [-Werror,-Wexplicit-ownership-type]
- (instancetype)initWithError:(NSError **)error {
                                          ^
```
I will add the `__autoreleasing` tag to the header.

Reply:
Done